docs/quickstart/csharp.md

```diff
@@ -199,23 +208,30 @@ The Grpc.Tools NuGet package contains the protoc and protobuf C# plugin binaries
 
 ### Obtaining the Grpc.Tools NuGet package
 
-#### Using Visual Studio, Xamarin Studio, or Monodevelop IDEs
+#### Using Visual Studio
 
 This example project already depends on the `Grpc.Tools.{{ site.data.config.grpc_release_branch | remove_first: "v" }}` NuGet package, so it should be included in `examples/csharp/helloworld/packages` when the `Greeter.sln` solution is built from your IDE, 
 or when you restore packages via `/path/to/nuget restore` on the command line.
 
-#### Using the .NET Core SDK
-
-From the `examples/csharp/helloworld-from-cli` directory:
+#### If you have a NuGet client that is __not__ at version 2.12
 
 ```
-> mkdir packages
-> cd packages
-> /path/to/nuget install Grpc.Tools
+$ mkdir packages && cd packages
+$ /path/to/nuget install Grpc.Tools
 ```
 
-* Note that you may have to update your NuGet executable to the latest version
-  in order to install the Grpc.Tools package, depending on your operating system.
+#### If you have a NuGet client that is at version 2.12
+
+NuGet 2.12 does not install the files from the Grpc.Tools package necessary on Linux and OS X.
+Without changing the version of NuGet that you're using, a possible workaround to obtaining the binaries included in the Grpc.Tools package 
+is by simply downloading the NuGet package and unzipping without a NuGet client, as follows.
```

Comment:
If you don't want to change the version of NuGet that you're using, a possible workaround to get these files is to download the NuGet package and unzip without a NuGet client, as follows.